src/runtime/wasm.ts

```diff
@@ -44,7 +44,7 @@ function jsonBaseType(type) {
 }
 
 // Returns a JSON representation that protobufjs can use to de/serialize entity data as protobufs.
-function toProtoJSON(schema: Schema) {
+export function toProtoJSON(schema: Schema) {
```

Comment:
Is @mykmartin good with this? It looks like `wasm.ts` aims to only export `EntityProtoConverter`, maybe he'd prefer this to be exported as a method on that class? Can do as needed in separate PR since he's already approved.

Reply:
This is fine. I expect the shape of this code to change a lot as the wasm implementation progresses, so I'm not worried about specifics yet.